design-documents/graph-ql/directives.graphqls

```diff
@@ -37,3 +37,9 @@ directive @typeResolver(class: String="") on UNION
     | OBJECT
 
 directive @cache(cacheIdentity: String="" cacheable: Boolean=true) on QUERY
+
+enum MutationResultStatus {
+    SUCCESS
+    FAILURE
+    PARTIAL_SUCCESS
```

Comment:
other words we can use: recoverable failure/error, 
if PARTIAL_SUCCESS then can we have a total success, total failure?
there's a REST & HTTP counterpart for this: 202: Accepted, 207 Multi-Status

there are IEEE papers on this : https://ieeexplore.ieee.org/document/5216347

Reply:
Multi-Status instead of Partial success?